drivers/ethernet/eth_stm32_hal_priv.h

```diff
@@ -40,7 +40,10 @@ struct eth_stm32_hal_dev_data {
 	struct device *clock;
 	struct k_mutex tx_mutex;
 	struct k_sem rx_int_sem;
-	K_KERNEL_STACK_MEMBER(rx_thread_stack,
+#ifdef CONFIG_SOC_SERIES_STM32H7X
+	struct k_sem tx_int_sem;
+#endif /* CONFIG_SOC_SERIES_STM32H7X */
+	K_THREAD_STACK_MEMBER(rx_thread_stack,
```

Comment:
Should be K_KERNEL_STACK_MEMBER

Reply:
You're right my bad, I'm updating the PR, fixing push available now 👍